front/src/common/redux/reducers/index.js

```diff
@@ -0,0 +1,29 @@
+const initialState = {
+    isLoad: false,
+    res: 0,
+};
+
+const reducer = (state = initialState, action) => {
+    switch (action.type) {
+        case "APP_LOAD":
```

Comment:
Please, move the action types to another file and export constants from there in the format
APP_LOAD = 'APP_LOAD'

Reply:
done